### DISCUSSION TITLE
Allow setting icons for scripts & commands

### DISCUSSION THREAD

**Discussion by YousufSSyed** [johnlindquist/kit#1410] [Ideas]
Icons can help distinguish each item when searching.

**Comment by johnlindquist** [comment]
In scripts, you can use:

```js
// Image: /Users/johnlindquist/.kit/logos/egghead/eggo.svg
```

In choices:

```js
await arg("", [
  {
    name: "egghead",
    img: "/Users/johnlindquist/.kit/logos/egghead/eggo.svg",
    value: "https://egghead.io",
  },
])
```

I'll look into bundling something like `react-icons` so we can add common icons by name, but for now, it needs to be a full path.
